EmbedCases.cmake

```diff
@@ -50,4 +50,9 @@ endmacro (opm_pack_stone)
 opm_pack_stone (upscale_relperm_benchmark)
 opm_pack_case (upscale_relperm_benchmark benchmark20)
 
-add_custom_target(benchmarks DEPENDS upscale_relperm_benchmark)
+if(INSTALL_BENCHMARKS)
+	add_custom_target(benchmarks ALL DEPENDS upscale_relperm_benchmark)
```

Comment:
I do not understand the logic here: add `benchmarks` to `ALL` (as opposed to in the else branch), then remove it from `ALL` again?